lib/ruby_lsp/requests/support/common.rb

```diff
@@ -9,14 +9,14 @@ module Common
         # or extensions by created by developers outside of Shopify, so be cautious of changing anything.
         extend T::Sig
 
-        sig { params(node: SyntaxTree::Node).returns(Interface::Range) }
+        sig { params(node: YARP::Node).returns(Interface::Range) }
         def range_from_syntax_tree_node(node)
           loc = node.location
 
           Interface::Range.new(
             start: Interface::Position.new(
               line: loc.start_line - 1,
-              character: loc.start_column,
```

Comment:
I had to adjust the start column by 1 but this may not be the right approach:

**Syntax Tree**

(rdbg) node
(tstring_content "foo/")
(ruby) node.location
#<SyntaxTree::Location:0x000000010c070c80
 @end_char=20,
 @end_column=20,
 @end_line=1,
 @start_char=16,
 @start_column=16,
 @start_line=1>

**YARP**

(rdbg) node
StringNode(15...21)((15...16), (16...20), (20...21), "foo/")
(ruby) node.location.slice
"\"foo/\""
(ruby) node.location.start_column
15
(ruby) node.location.end_column
20